matchzoo/preprocessor/process_units.py

```diff
@@ -228,3 +228,70 @@ def fit(self, tokens: list, ngram: int=3):
         """
         _, input_dim = self._create_n_letters(tokens, ngram)
         self._state = {'input_dim': input_dim}
+
+
+class VocabularyUnit(StatefulProcessorUnit):
+    """
+    Vocabulary class.
+
+    Examples:
+        >>> vocab = VocabularyUnit()
+        >>> vocab.fit(['A', 'B', 'C', 'D', 'E'])
```

Comment:
vocab.fit(['hello world', 'python', 'machine learning']), is that right?

Reply:
the idea is `hello word` should be transformed to `hello` and `word` by TokenizeUnit.

Reply:
The input of `TokenizeUnit` is raw text, while input for `VocabularyUnit` is list of tokens.